### ISSUE TITLE
Make the list of pages publicly available.

### ISSUE THREAD

**Issue by klntsky** [lindylearn/aboutideasnow#19]
I couldn't find the list anywhere on the website. By the way, the search does not work anymore.

**Comment by zft9xgy** [comment]
Yep, i can confirm search does not work today for me, i trying different browser and didnt work. 

Public list of the websites will be really nice featured. 

Thank you for your project, I found it very inspiring and interesting, have you thought about making a Spanish version?
